### PR TITLE
FUF 181 - send query params to backend

### PR DESCRIPTION
**Pull request by mlpinit**
# What

* Send query params to backend

# Why

* This allows us to send a token and other optional params that may be required to distinguish between FUF and initial forms.

# Clarification
When query params are supplied, we send them to the backend. This allows
us to authenticate a user and ensure that we know how to handle a
request coming from an embeded form.

cf: https://github.com/dobtco/formrenderer-base/issues/181

### CODE REVIEW

File: src/helpers/query_params.coffee [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FormRenderer.queryParams = (value) ->
```
Comment (mlpinit) [inline src/helpers/query_params.coffee]: FYI: I tried to use https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams instead but it looks like the browser version we're currently targeting does not support it.

File: src/main.coffee [hunk 2]
```diff
@@ -206,6 +211,8 @@ window.FormRenderer = FormRenderer = Backbone.View.extend
         skip_validation: @options.skipValidation
       },
       @options.saveParams,
+      # TODO(fuf): We should exclude the follow up form params from here as they
+      # are already part of the loadParams. This may be a breaking change.
```
Comment (jessicb) [inline src/main.coffee]: Does including them twice change the payload that is sent at all?

Comment (mlpinit) [reply src/main.coffee]: We have the follow up form params included at both the`follow_up_form_params` level and the root level. You made me notice that the `follow_up_form_params` scoping might be overdoing it here. Right now we only have the `follow_up_form_id` and `initial_response_id` under follow up form params and those won't conflict with any other load params or save params. I'll remove that scope and then I can remove the `@followUpFormParams` duplicate call.

Comment (jessicb) [reply src/main.coffee]: Cool, the scope changes look good.

**Pull request merged**